Fix #2570 : Fixed Typos and Errors

riturajjain2000 opened the pull request:
<!--
  - Thanks for submitting code to Oppia! Please fill out the following as part of
  - your pull request so we can review your code more easily.
  -->

## Explanation
Fixes #2570 
Fixed Typos and errors in 3 Files

## Checklist
<!-- Please tick the relevant boxes by putting an "x" in them. -->
- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

riturajjain2000 committed:
Fixed Typos #2570

riturajjain2000 committed:
Fixed Typos #2570

FareesHussain reviewed (approved):
LGTM, thanks

rt4914 reviewed (changes requested):
While we are fixing this lets replace `softinputkeyboard ` with `SoftInputKeyboard`.

riturajjain2000 committed:
Update KeyboardHelper.kt

riturajjain2000 committed:
Update KeyboardHelper.kt

riturajjain2000 commented:
> While we are fixing this lets replace `softinputkeyboard ` with `SoftInputKeyboard`.

Done!

rt4914 reviewed (approved):
LGTM, thanks.

riturajjain2000 commented:
> LGTM, thanks.

Can this be merged ?

MaskedCarrot commented:
@riturajjain2000 please wait for @BenHenning review on this.

riturajjain2000 commented:
> @riturajjain2000 please wait for @BenHenning review on this.

Ok sure.

BenHenning commented on app/src/main/java/org/oppia/android/app/recyclerview/StartSnapHelper.kt:
```diff
@@ -5,6 +5,7 @@ import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.LinearSnapHelper
 import androidx.recyclerview.widget.OrientationHelper
 import androidx.recyclerview.widget.RecyclerView
+import kotlin.Throws
```
Why is this needed? It doesn't seem to be used. Suggest removing it.

FareesHussain replied:
@BenHenning Actually, I mentioned this in the issue. 
It is used in the below line not sure if this is needed 
https://github.com/oppia/oppia-android/blob/5fc69fffb97e0c8a0d529238926f3e078cf88b09/app/src/main/java/org/oppia/android/app/recyclerview/StartSnapHelper.kt#L18

FareesHussain replied:
@riturajjain2000 The PR reviewer can resolve the conversation after confirming the suggested changes are done or if anything left to discuss. The conversation should not be resolved by PR authors.

riturajjain2000 replied:
Ok ,I'll keep that in mind

BenHenning commented on app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt:
```diff
@@ -4,11 +4,11 @@ import android.content.Context
 import android.view.inputmethod.InputMethodManager
 
-/** KeyboardHelper helps to change the visibility of softinputkeybord. */
+/** KeyboardHelper helps to change the visibility of SoftInputKeyboard. */
```
``SoftInputKeyboard`` isn't a class. Suggest writing it out, instead: ``the soft input keyboard``. Ditto below.

BenHenning reviewed (approved):
Thanks @riturajjain2000! Had two comments. The change in my codeowner file LGTM, but @rt4914 will need to approve the other two files.

BenHenning reviewed (changes requested):
Ah it looks like @rt4914's approval was cleared, but it's still considered approved. Switching my review to 'request changes' for the open comments.

riturajjain2000 committed:
fix typos and optimised imports

riturajjain2000 committed:
fix typos and optimised imports

BenHenning commented on app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt:
```diff
@@ -4,11 +4,11 @@ import android.content.Context
   companion object {
     /**
-     * This method hides softinputkeybord
+     * This method hides the soft input keyboard
      * @param view is the input view
      * @param context context of the activity
```
```suggestion
     * This method hides the soft input keyboard.
     *
     * @param view the input view
     * @param context the activity context
```

BenHenning commented on app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt:
```diff
@@ -20,7 +20,7 @@ class KeyboardHelper {
 
     /**
-     * This method shows softinputkeybord
+     * This method shows the soft input keyboard
      * @param view is the input view
      * @param context context of the activity
```
```suggestion
     * This method shows the soft input keyboard.
     *
     * @param view the input view
     * @param context the activity context
```

BenHenning reviewed (approved):
Thanks @riturajjain2000. Added some opportunistic suggested fixes. Once addressed, this PR LGTM.

BenHenning reviewed (changes requested):
Actually, requesting changes to block submission until the last two comments are addressed (just to make sure it doesn't get accidentally merged).

riturajjain2000 committed:
Merge remote-tracking branch 'upstream/develop' into fix-typos

riturajjain2000 committed:
addressed changes

riturajjain2000 committed:
addressed changes

riturajjain2000 commented:
> Actually, requesting changes to block submission until the last two comments are addressed (just to make sure it doesn't get accidentally merged).

Made changes as suggested

FareesHussain committed:
Fix part of #632: Replace LanguageSelectionAdapter with BindableAdapter (#2392)

* issue 2098

* test commit

* Update gradle.xml

* Update misc.xml

* Fix gradle.xml

* initial inmplementation

* Ready for Review

* Updated BAZEL with new viewmodels

* Reformatting code and making changes in other configuration

* Fixed the Tablet issue

* Adding Requested changes

* Resolving requested changes

* NIT fix

* Resolving requested changes

* NIT fix

* NIT fix

* update fun name

* using databinding for audio language selection

* using databinding for app language selection

* ktlint fix

* Fix bazel check

* Resolving Requested changes

* Nit fix

* Radio Button fix

* NIT fix

* Using lifecycle owner to update the radio button

* Nit fix

yurilev committed:
Fix #2504: Address nit for HomeActivityTest to use verifyHomeRecyclerViewHasGridColumnCount() helper (#2586)

* Use the `verifyHomeRecyclerViewHasGridColumnCount` helper in one applicable test where it isn't used yet.

* Fix typo.

* Remove incorrect/confusing comment.

What is actually being verified is the number of columns. In this instance, it's true that there are also two rows, but none of the similar tests check the number of rows, so I assume the reference was unintentional.

* Empty commit to retrigger CI tests.

priyanka0906 committed:
Fix #2462: Optimised TextInputLayout in admin_pin_activity (#2598)

* rtl support to topic lessons tab

* rtl support to topic lessons tab

* rtl support to topic lessons tab

* rtl support to topic lessons tab

* rtl support to topic lessons tab

* Update Project.xml

* Update Project.xml

* Update Project.xml

* Update Project.xml

* Update Project.xml

* rtl support to topic lessons tab

* rtl support to topic lessons tab

* commit 1

* optimized TextInputLayout

* optimized TextInputLayout

* optimized TextInputLayout

* optimized TextInputLayout

* optimized TextInputLayout

* Update example.proto

* Update example.proto

Co-authored-by: priyanka0906 <priyank02amb@rediffmail.com>

MaskedCarrot committed:
Fix #2595: Migrate to ViewPager2 in layout/my_downloads_fragment.xml (#2597)

* migrated MyDownloadsFragmentPresenter.kt to viewPager2

* migrated MyDownloadsFragmentPresenter.kt to viewPager2

KartikeySharma committed:
Formatted XML files (#2582)

khushal1707 committed:
Fix #2529: Fix username's cutoff in Various screens (#2556)

* Fix#2529: Fix username's cutoff in Various screens

* Requested changes

* Changes in profile list's profile view

* Changes in Profile Chooser screen reverted

* Requested Changes

viktoriias committed:
Fix #2132: Chapter List Page Issues (#2232) - Blur and lock thumbnails of inactive cards. (#2422)

* Lock and blur

* Replace color/white with #FFFFFF

* Add \n

* Move isBlurred to LessonThumbnailImageView.

* Remove unused import.

* Move dependency on Glide away from from ImageLoader.

* Remove unneeded line.

* Refactor Transformation-related code, add KDocs.

* Renaming.

* Remove unused import.

* Reformat.

* Add Mockk test

* Add Mockk test - lint

* Add blurring tests.

* Mock ImageLoader in StoryActivityTest.

* Add delay before image loading.

* Nits and improvements.

* Remove unused import.

* Remove unused import.

* Set image resources in LessonThumbnailImageView in addition to calling GlideImageLoader.

* BlurTransformation re-write.

* BlurTransformation re-write.

* Add TestImageLoaderModule to some tests.

* Lint

* Add TestImageLoaderModule to more tests.

* Add TestImageLoaderModule to more tests.

* Update utility/src/main/java/org/oppia/android/util/parser/BlurTransformation.kt

Co-authored-by: Ben Henning <henning.benmax@gmail.com>

* Update utility/src/main/java/org/oppia/android/util/parser/BlurTransformation.kt

Co-authored-by: Ben Henning <henning.benmax@gmail.com>

* TestGlideImageLoader Refactor.

* Reformat.

* Reformat.

* Reformat.

* Nits and comments.

* Revert changes in landscape design.

* Change BlurTransformation to return a copy instead of 'toTransform'.

* Update app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt

Co-authored-by: Ben Henning <henning.benmax@gmail.com>

* Refactor TestGlideImageLoader.

* Nits

Co-authored-by: Ben Henning <henning.benmax@gmail.com>

Arjupta committed:
Fixes part of #2258: Changes in Add Profile Activity[a11y] (#2456)

* Fix #2258: Changes in Add Profile Activity[a11y]

Fixes Issue 3 of #2258, Changed the button to diabled according to the
state and TextColor as per Chantel's suggesstion

* Using color resource instead of hardcoded color

* Same changes applied to other orientation files

prayutsu committed:
Fix #1873: Fix OnboardingFragmentTest test cases on Robolectric (#2482)

* fixed one test

* fix 4 remaining tests

* try fixing tests bu introducing scrollview

* tried adding scrollview

* fix AndroidManifest.xml

* fix nit

* fix basic set of tests

* fixed 3 failing tests

* removed extra changes

* removed extra changes

* try fixing tests with testCoroutineDispatchers

* try advancedUntilIdle()

* tried migrating to viewpager2

* tried implementing new methods

* Replace ViewPager with ViewPager2

* Add ignore annotations

* fix lint

* add dependencies to BUILD.bazel

* remove unnecessary ignore annotations

* fix lint

* fix lint

* fix lint

* fix lint error

* remove redundant lines

* Fix Viewpager2 regressions

* Fix lint errors.

* Remove comments

* Fix all test cases on Robolectric

* Try fixing flakiness in espresso

* Revert "Try fixing flakiness in espresso"

This reverts commit 2151c455

* Replace isDisplayed() with isCompletelyDisplayed()

* Fix suggested changes

* Try fixing test cases by using isADescendantOf()

* Revert changes

* Try stabilizing ttion est cases by changing configuration

* Revert last commit to remove unwanted configuration changes

* Fix ktlint error

* Try adding custom ViewMatcher

* Add custom ViewMatcher

* Add custom ViewAction for scrolling ViewPager pages

* Disable smooth scrolling to fix test cases

* Remove holder.setIsRecycleable(false) and optimise perform()

* Remove unnecessary id of constraintLayout in onboarding_slide.xml

* Replace advancedUntilIdle bu runCurrent wherever applicable

* Remove unnecessary delay

* Add argument comment for smoothScroll

* Add a space after argument comment

* Fix suggested changes.

rt4914 reviewed (approved):
LGTM, thanks.

rt4914 commented:
@riturajjain2000 I have restarted the tests, if they fail again, just update your branch with latest develop and it should work.

BenHenning commented on app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt:
```diff
@@ -4,13 +4,14 @@ import android.content.Context
-     * This method hides softinputkeybord
-     * @param view is the input view
-     * @param context context of the activity
+     * This method hides the soft input keyboard
```
```suggestion
     * This method hides the soft input keyboard.
```

riturajjain2000 replied:
> 

I addressed all the changes in a new PR. PTAL

BenHenning commented on app/src/main/java/org/oppia/android/app/utility/KeyboardHelper.kt:
```diff
@@ -20,9 +21,10 @@ class KeyboardHelper {
-     * This method shows softinputkeybord
-     * @param view is the input view
-     * @param context context of the activity
+     * This method shows the soft input keyboard
```
```suggestion
     * This method shows the soft input keyboard.
```

riturajjain2000 replied:
I addressed all the changes in a new PR. PTAL

BenHenning reviewed (changes requested):
Thanks @riturajjain2000! Couple of things:
1. Please make sure to fully read comments before considering them addressed (e.g. in this case you missed the '.' in my original suggested code). For suggested code changes, it's also fine just to commit that code directly to your branch.
2. Please always respond to comment threads saying you've addressed them, otherwise the assumption is that you might not of. We require all discussion threads to be addressed before a PR will be merged.

Prince-kushwaha committed:
"Fix #2580: Reformatted the files (#2584)

* reformatted the files

* Update the previous commit

* revert changes in misc.xml

riturajjain2000 commented:
> Thanks @riturajjain2000! Couple of things:
> 
> 1. Please make sure to fully read comments before considering them addressed (e.g. in this case you missed the '.' in my original suggested code). For suggested code changes, it's also fine just to commit that code directly to your branch.
> 2. Please always respond to comment threads saying you've addressed them, otherwise the assumption is that you might not of. We require all discussion threads to be addressed before a PR will be merged.

OK , I'll keep this in mind from now on.

riturajjain2000 committed:
Merge branch 'fix-typos' of https://github.com/riturajjain2000/oppia-android into fix-typos

riturajjain2000 committed:
addressed changes
